app/scripts/lieferplanungen/detail/korbplanung.html

```diff
@@ -94,8 +94,8 @@ <h3 class="panel-title lieferplanung-title">
               </div>
             </span>
             <span class="pull-right">
-              <small class="lieferplanung"><span ng-show="projekt.preiseSichtbar"><b><span title="{{getDurchschnittspreisInfo(abotypLieferung)}}">Ø {{getDurchschnittspreis(abotypLieferung)|ooCHF:true}}</span></b> | </span>{{abotypLieferung.datum | date}}</small>
-              <span ng-show="valuesEditable()" class="btn btn-default btn-sm"
+              <small class="lieferplanung"><span ng-show="projekt.preiseSichtbar"><b><span title="{{getDurchschnittspreisInfo(abotypLieferung)}}">Ø {{getDurchschnittspreis(abotypLieferung)|ooCHF:true:projekt.waehrung}}</span></b> | </span>{{abotypLieferung.datum | date}}</small>
+              <span class="btn btn-default btn-sm"
```

Comment:
Didn't you introduce this 'ng-show="valuesEditable()' in v2.2?

Reply:
done